indico/web/templates/layout/base.html

```diff
@@ -1,36 +1,38 @@
 {% block page %}
     <div class="{% block page_class %}management-page{% endblock %}"
          {% if self.page_id()|trim %}id="{% filter trim %}{% block page_id %}{% endblock %}{% endfilter %}"{% endif %}>
-        {% if self.title() %}
+        {% if self.title() or self.description() or self.info() %}
```

Comment:
I wanted it so I can have this layout (page description):
![image](https://cloud.githubusercontent.com/assets/1579899/21115360/48cd7006-c0b1-11e6-817c-3f15f6b85367.png)